po/sync-monitor.pot

```diff
@@ -1,115 +1,150 @@
 # SOME DESCRIPTIVE TITLE.
-# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
-# This file is distributed under the same license as the PACKAGE package.
+# Copyright (C) YEAR Canonical Ltd.
+# This file is distributed under the same license as the sync-monitor package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: sync-monitor\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-09-10 18:43+0000\n"
+"POT-Creation-Date: 2020-02-20 12:20-0600\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
 "Language: \n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: ../src/sync-account.cpp:739
-msgid "Failed to configure account"
+#: authenticator/main.qml:153
+msgid "Account sign-in failed. Please select your account to sign back in."
 msgstr ""
 
-#: ../src/sync-account.cpp:748
-msgid "Forbidden / access denied"
+#: authenticator/main.qml:60 authenticator/main.qml:78
+msgid "Accounts"
 msgstr ""
 
-#: ../src/sync-account.cpp:750
-msgid "Object not found / unassigned field"
+#: src/sync-daemon.cpp:628
+msgid "An account failed to sync. Would you like to sign in again?"
 msgstr ""
 
-#: ../src/sync-account.cpp:752
+#: src/sync-account.cpp:748
 msgid "Command not allowed"
 msgstr ""
 
-#: ../src/sync-account.cpp:755
-msgid "Proxy authentication required"
+#: src/sync-account.cpp:770
+msgid "Connection certificate has expired"
 msgstr ""
 
-#: ../src/sync-account.cpp:757
-msgid "Disk full"
+#: src/sync-account.cpp:772
+msgid "Connection certificate is invalid"
 msgstr ""
 
-#: ../src/sync-account.cpp:759
-msgid "Failed to sync due to a remote problem"
+#: src/sync-account.cpp:768
+msgid "Connection timeout"
```

Comment:
```suggestion
msgid "Connection timed out"
```
Avoids this issue https://grammarist.com/spelling/timeout-time-out/
https://www.dictionary.com/browse/timeout